config/tutorials/hdfs-logs/searcher-1.yaml

```diff
@@ -1,5 +1,7 @@
 version: 0
 node_id: searcher-1
+metastore_uri: s3://path/to/bucket/indexes # change this to your S3 bucket
```

Comment:
I prefer to commit only things that can work out of the box. 
We can add `metastore_uri` and `default_index_root_uri` in comment.

Reply:
I think, my proposition is that to even get rid of these files. what do you think? right now it's not used in the tutorial

Reply:
That's why I proposed their removal but it seems we have found a use case for them in the local hdfs cluster.

Reply:
Yes exactly :)